src/Fantomas.Tests/ListTests.fs

```diff
@@ -403,8 +403,8 @@ let ``line comment inside array`` () =
  |]
 """
 
+#if RELEASE
```

Comment:
does this really work? I ask because I've found myself in the past that yes, when building in Debug mode, the DEBUG symbol is defined, but the same is not true for Release mode, RELEASE symbol is not defined unless you do this manually (or use `#if !DEBUG` instead). Or maybe this was my findings in .NETFramework but it is different in .NETCore/.NET5 now?

Reply:
![image](https://user-images.githubusercontent.com/2621499/111114811-11fdc680-8564-11eb-8118-903cec32d724.png)